content/ngf/overview/gateway-api-compatibility.md

```diff
@@ -124,10 +123,9 @@ See the [static-mode]({{< ref "/ngf/reference/cli-help.md#static-mode">}}) comma
     - `Accepted/False/Invalid`
     - `Accepted/False/UnsupportedValue`: Custom reason for when a value of a field in a Gateway is invalid or not supported.
     - `Accepted/False/GatewayConflict`: Custom reason for when the Gateway is ignored due to a conflicting Gateway.
-          NGINX Gateway Fabric only supports a single Gateway.
     - `Programmed/True/Programmed`
     - `Programmed/False/Invalid`
-    - `Programmed/False/GatewayConflict`: Custom reason for when the Gateway is ignored due to a conflicting Gateway. NGINX Gateway Fabric only supports a single Gateway.
+    - `Programmed/False/GatewayConflict`: Custom reason for when the Gateway is ignored due to a conflicting Gateway.
   - `listeners`
```

Comment:
You're missing a piece of the acceptance criteria in the ticket. Relating to Gateway status address.